projects/packages/videopress/src/class-wpcom-rest-api-v2-endpoint-videopress.php

```diff
@@ -172,6 +172,36 @@ public function register_routes() {
 				},
 			)
 		);
+
+		// VideoPress Private Setting Route.
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/site-privacy-setting',
```

Comment:
Is this endpoint intended for Simple sites as well? If not you could consider registering it in a separate file, under perhaps `videopress/v1`.

Reply:
Luiz, Foteini is right. We don't need this endpoint in the block editor context, meaning for Simple sites, either.